kitsune/users/tasks.py

```diff
@@ -20,10 +19,9 @@
 def process_event_delete_user(event_id):
     event = AccountEvent.objects.get(id=event_id)
 
-    anonymize_user(event.profile.user)
-
-    event.status = AccountEvent.PROCESSED
-    event.save()
+    if delete_user_pipeline.delay(event.profile.user):
```

Comment:
Is the `.delay(...)` leftover from an earlier iteration? The `delete_user_pipeline` function is not a task. Or perhaps it needs to be pulled into this file and made into a Celery task?